pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -138,3 +150,23 @@ def get_fort_in_range(self):
             return fort
 
         return None
+
+    def get_fort_details(self, fort_id, latitude, longitude):
```

Comment:
Thanks for the feedback. I will work on this further tomorrow.

On 29 Jul 2016 18:54, "Eli White" <notifications@github.com> wrote:

> In pokemongo_bot/cell_workers/seen_fort_worker.py
> <https://github.com/PokemonGoF/PokemonGo-Bot/pull/1671#discussion_r72833653>
> :
>
> > @@ -138,3 +150,23 @@ def get_fort_in_range(self):
> >              return fort
> >
> >          return None
> > +
> > +    def get_fort_details(self, fort_id, latitude, longitude):
>
> This method that does caching probably shouldn't be here. We have more
> than one place that looks at fort information, so we should save that data
> and provide a function that everything can use. We should pull this out
> into *init*.py, or ideally a new file where we can put methods like this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PokemonGoF/PokemonGo-Bot/pull/1671/files/a8df1799bd46d1df9789666fef34d27ce4feac99#r72833653>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABg2CJjJNC5LdtsfBi8IydD6poJGw-t2ks5qaj5QgaJpZM4JYaNI>
> .
>